hackathon_site/dashboard/frontend/src/slices/hardware/cartSlice.ts

```diff
@@ -44,12 +45,15 @@ const cartSlice = createSlice({
         removeFromCart: (state, { payload }: PayloadAction<number>) => {
             cartAdapter.removeOne(state, payload);
         },
+        updateCart: (state, { payload }: PayloadAction<Update<CartItem>>) => {
```

Comment:
Didn't know about the `Update` type, cool! Kind of a gross syntax for actually doing the update, but who am I to argue with the redux gods

Reply:
Yeah lol, I wasn't sure the correct syntax to use the `Update` type, but it's magic seems to be working behind the scenes.